userspace/falco/grpc_server.cpp

```diff
@@ -211,7 +224,7 @@ void falco::grpc::server::run()
 
 	while(server_impl::is_running())
 	{
-		sleep(1);
+		std::this_thread::sleep_for(std::chrono::milliseconds(100));
```

Comment:
I can see why we have a performance increase now :)

Reply:
I can't see how this line can be related to that.